app/gui/docs/product/shortcuts.md

```diff
@@ -139,3 +139,4 @@ broken and require further investigation.
 | <kbd>ctrl</kbd> + <kbd>shift</kbd> + <kbd>arrow up</kbd>           | Pop a breadcrumb without navigating.                                                                                           |
 | <kbd>cmd</kbd> + <kbd>i</kbd>                                      | Reload visualizations. To see the effect in the currently shown visualizations, you need to switch to another and switch back. |
 | <kbd>ctrl</kbd> + <kbd>shift</kbd> + <kbd>b</kbd>                  | Toggle read-only mode.                                                                                                         |
+| <kbd>ctdl</kbd> + <kbd>shift</kbd> + <kbd>u</kbd>                  | Dump the suggestion database as JSON to the console. Available only in debug mode, and only if the component browser is open.  |
```

Comment:
```suggestion
| <kbd>ctrl</kbd> + <kbd>shift</kbd> + <kbd>u</kbd>                  | Dump the suggestion database as JSON to the console. Available only in debug mode, and only if the component browser is open.  |
```

Reply:
Looks like a useful diagnosting tool. I have always wondered how to debug content of the suggestion DB?